Sample Code/Time of Flight/Readme.md

```diff
@@ -2,30 +2,11 @@
 
 This is a sample of JavaScript code used to retrieve information from the time of flight sensors via WebSockets. There are four time of flight sensors on Misty I: left, center, right, and back. The left, center, and right sensors are all located in the front. This sample code subscribes to the center time of flight sensor only.
```

Comment:
Shouldn't it be, "There are four horizontal time of flight sensors on Misty II:"

Reply:
I use `Range` and `Edge`  in the docs (that's what they're called in ToF event messages). Clearly this README didn't get the memo. Will update.